src/gui/properties/trackerlistwidget.cpp

```diff
@@ -283,24 +283,31 @@ void TrackerListWidget::clear()
 
 void TrackerListWidget::loadStickyItems(const BitTorrent::TorrentHandle *torrent)
 {
-    QString working = tr("Working");
-    QString disabled = tr("Disabled");
+    const QString working {tr("Working")};
+    const QString disabled {tr("Disabled")};
+    const QString torrentDisabled {tr("Disabled for this torrent")};
 
     // load DHT information
-    if (BitTorrent::Session::instance()->isDHTEnabled() && !torrent->isPrivate())
+    if (BitTorrent::Session::instance()->isDHTEnabled() && !torrent->isPrivate() && !torrent->isDHTDisabled())
         m_DHTItem->setText(COL_STATUS, working);
+    else if (torrent->isDHTDisabled())
+        m_DHTItem->setText(COL_STATUS, torrentDisabled);
     else
         m_DHTItem->setText(COL_STATUS, disabled);
```

Comment:
>Why not print `Disabled for this torrent` in case of **private** torrents?

No reason. I just hope people won't be confused when they see it and think they can enable DHT etc for private torrents. But then again the same can be said about the global setting. Changed.

>Or if it's important to see that DHT is disabled for this particular torrent even when it is globally disabled:

It is important because the global setting is irrelevant if it's disabled for the torrent.